src/sbl_filing_api/services/validators/filing_validators.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from sbl_filing_api.entities.models.dao import FilingPeriodDAO, FilingDAO
+from .base_validator import ActionValidator
+
+log = logging.getLogger(__name__)
+
+
+class CheckPeriodExists(ActionValidator):
+    def __init__(self):
+        super().__init__("check_period_exists")
+
+    def __call__(self, period: FilingPeriodDAO, period_code: str, **kwargs):
+        if not period:
+            return f"The period ({period_code}) does not exist, therefore a Filing can not be created for this period."
+
+
+class CheckPeriodFilingExists(ActionValidator):
+    def __init__(self):
+        super().__init__("check_period_filing_exists")
```

Comment:
maybe this one should be something like period filing not started; at the moment it has the same naming convention as period exists above, but it's the opposite logic as what's above.

Reply:
The point is to check if a filing already exists for that period and if it does fail.  So the two checks when starting a filing are:
- does the period exist, if not fail
- does a filing already exist for that period, if so fail

I had to name it something different from the sign "check_filing_exists" which has the reverse logic, it fails to sign if a the filing for that period + LEI doesn't exist yet.  But since all the validators get put into one registry, I needed a different name (key) in the registry map.  Naming is definitely something an issue we could run up against and is something I thought that maybe sticking all the validator subs into one file might help, but then that could potentially get very large.  But I also though of moving your signing "check_filing_exists" into this filing_validators.py and naming it like "check_filing_not_exists" and have this one be "check_filing_exists".